src/components/Article.jsx

```diff
@@ -1 +1,17 @@
-// export const Article = () => ();
+import React from 'react';
+
+const Article = ({ title, text, date }) => (
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+    <time dateTime="2019-08-30" data-cy="date">
+      {date.toString()}
+    </time>
```

Comment:
```suggestion
    <time dateTime={dateISO} data-cy="date">
      {dateLocaleString}
    </time>
```
The problem with time is still not fixed. Please add these variables to your component and make sure that you don't have any errors in your project
```
const dateISO = today.toISOString().slice(0, 10);
const dateLocaleString = today.toLocaleDateString('en-US', {
  year: 'numeric',
  month: 'long',
  day: 'numeric',
});
```